### PR TITLE
[TEST](bang-ops):add diff_iou_rotated_sort_vertices_forward api test

### PR DESCRIPTION
**Pull request by pingjiang123**
Thanks for your contribution and we appreciate it a lot. 

## 1. Motivation

Please describe your motivation and the goal you want to achieve through this pull request.

## 2. Modification

Please briefly describe what modification is made in this pull request, and indicate where to make the modification.

### CODE REVIEW

**Review by DanieeelLiu — APPROVED** [review]

File: bangc-ops/test/mlu_op_gtest/api_gtest/src/gtest/diff_iou_rotated_sort_vertices_forward/diff_iou_rotated_sort_vertices_forward_general.cpp [hunk 1]
```diff
@@ -0,0 +1,518 @@
...
+                                         4, std::vector<int>({4, 17, 9})},
+                        MLUOpTensorParam{MLUOP_LAYOUT_ARRAY, MLUOP_DTYPE_INT32,
+                                         3, std::vector<int>({4, 16, 8})}),
+        testing::Values(MLUOP_UNKNOWN_DEVICE),
+        testing::Values(MLUOP_STATUS_BAD_PARAM)));
+
+INSTANTIATE_TEST_CASE_P(
+    vertices_error_layout, diff_iou_rotated_sort_vertices_forward_general,
+    testing::Combine(
+        testing::Values(MLUOpTensorParam{MLUOP_LAYOUT_NCHW, MLUOP_DTYPE_FLOAT,
```
Comment (GuangpingYuan) [inline bangc-ops/test/mlu_op_gtest/api_gtest/src/gtest/diff_iou_rotated_sort_vertices_forward/diff_iou_rotated_sort_vertices_forward_general.cpp:384]: 支持的是array，给NCHW也会有防呆吗？ARRARY且没特殊含义的情况下一般不强求有防呆。

File: bangc-ops/test/mlu_op_gtest/api_gtest/src/gtest/diff_iou_rotated_sort_vertices_forward/diff_iou_rotated_sort_vertices_forward_general.cpp [hunk 2]
```diff
@@ -0,0 +1,518 @@
...
+                                         3, std::vector<int>({4, 16, 24})}),
+        testing::Values(MLUOpTensorParam{MLUOP_LAYOUT_ARRAY, MLUOP_DTYPE_INT32,
+                                         2, std::vector<int>({4, 16})}),
+        testing::Values(MLUOpTensorParam{MLUOP_LAYOUT_ARRAY, MLUOP_DTYPE_INT32,
+                                         4, std::vector<int>({4, 16, 9})}),
+        testing::Values(MLUOP_UNKNOWN_DEVICE),
+        testing::Values(MLUOP_STATUS_BAD_PARAM)));
+
+INSTANTIATE_TEST_CASE_P(
+    mask_error_layout, diff_iou_rotated_sort_vertices_forward_general,
```
Comment (GuangpingYuan) [inline bangc-ops/test/mlu_op_gtest/api_gtest/src/gtest/diff_iou_rotated_sort_vertices_forward/diff_iou_rotated_sort_vertices_forward_general.cpp:396]: 对同一个tensor的不同检查可以写在同一个TEST_P里，可以组合出这几种错误的情况，这样代码量少，写起来比较简洁

**Comment by pingjiang123** [comment]
[       OK ] large_tensor_notsupported/diff_iou_rotated_sort_vertices_forward_general.negative/0 (0 ms)
[----------] 1 test from large_tensor_notsupported/diff_iou_rotated_sort_vertices_forward_general (0 ms total)

[----------] Global test environment tear-down
[==========] 40 test cases from 8 test suites ran. (1057 ms total)
[  PASSED  ] 40 test cases.

**Review by GuangpingYuan — APPROVED** [review]
LGTM

**Review by wangrt1 — APPROVED** [review]

**Pull request merged**